Server/src/tests/controllers/GraphsController.spec.ts

```diff
@@ -60,18 +60,18 @@ describe('SavedGraphs Controller ', () => {
     }
     mockRequest = {
       params: {
-        oneSavedGraph: '1'
+        graphStateId: '1'
       }
     }
-    await SavedGraphsController.createRequestForSingleSavedGraph(mockRequest as Request, mockResponse as Response)
+    await SavedGraphsController.createRequestForSingleGraph(mockRequest as Request, mockResponse as Response)
     expect(mockResponse.json).toBeCalledWith(expectedResponse);
     expect(mockResponse.status).toBeCalledWith(200);
   });
 
   //   test('Invalid GraphID Request', async () => {
   //     mockRequest = {
   //       params: {

```

Comment:
add a todo? on the test? or remove the comments

Reply:
Some tests broke after we switched to extracting JWT. These will be fixed since we dont yet know how to update our tests